automation/taskcluster/lib/tasks.py

```diff
@@ -645,10 +658,10 @@ def _craft_raptor_task(
                 )],
                 "command": command,
                 "env": {
-                    "EXTRA_MOZHARNESS_CONFIG": json.dumps({
+                    "EXTRA_MOZHARNESS_CONFIG": {'task-reference': json.dumps({
```

Comment:
I don't quite understand this: isn't the value of `task-reference` usually something like `"<thing>"`?

Reply:
👍 to this question. To pursue https://github.com/mozilla-mobile/fenix/pull/5257#discussion_r324563695, `task-reference` looks for "<thing>" within any string. 

Here, `json.dumps()` outputs a string and `apk_url` contains `<thing>` defined here https://github.com/mozilla-mobile/fenix/pull/5257/files#diff-59f90d509bc6ae2c9d877250c5ba49c5R622. So taskgraph finds `<thing>` in this big string and replaces it.